homeassistant/components/iqvia/__init__.py

```diff
@@ -37,21 +37,18 @@
 
 FETCHER_MAPPING = {
     (TYPE_ALLERGY_FORECAST,): (TYPE_ALLERGY_FORECAST, TYPE_ALLERGY_OUTLOOK),
-    (TYPE_ALLERGY_TODAY, TYPE_ALLERGY_TOMORROW): (
-        TYPE_ALLERGY_INDEX,),
+    (TYPE_ALLERGY_TODAY, TYPE_ALLERGY_TOMORROW): (TYPE_ALLERGY_INDEX,),
```

Comment:
I wouldn't including linting of a separate integration; it may be totally correct, but I'd rather handle that my next time through the integration – I'd hate for something to break here.

Reply:
To not break things are the integrations tested (when possible and it can be done without investing a lot of setup time) if dependencies are updated.

I would love to leave the style modifications to `black` and put this topic to bed but till this is happening, it's up to use to keep an eye on this. If one starts with something new based on the current code then during the review is pointed out that it needs to change. And this is frustrating for the contributors.

Reply:
I understand the frustration. Unfortunately, though, we _haven't_ standardized on, say, `black` yet (I happen to use the Chromium version of `yapf`).

Once we do standardize, this file – and all files – will be taken care of; until then, the only thing this does (in the context of this PR) is introduce unnecessary changes that may harm something unintentionally.

That's my $0.02. Ultimately, I trust your judgement and will leave it in your hands.